mojave/code/datums/ai/basic_mobs/basic_subtrees/speech_subtree.dm

```diff
@@ -3,3 +3,11 @@
 	emote_hear = list("lets out a weak groan.")
 	emote_see = list("idly twitches")
 	speak = list("hnnnhhh","heuhhhh","nnnnhhh")
+
+
+/datum/ai_planning_subtree/random_speech/ms13/robot
+	speech_chance = 10
+	emote_hear = list("Thrumms quietly with electrical current.")
+	emote_see = list("scans the surrounding area ")
+	speak = list("PROCESSING...","ENGAGING PATROL PROTOCOL","ROBCO PROVIDES FIRSTCLASS SERVICE WORLDWIDE.", "BUY A NUKA COLA...TODAY!", "ATMOSPHERIC RADIATION LEVEL UNSAFE FOR HUMAN CONSUMPTION. CONSUME CANNED HUMIDITY.", "YOUR SAFETY NOW PROVIDED BY THE GUYS AT ROBCO SERVICES", "WARNING: PLEASE KEEP PATENTED ROBCO TECHNOLOGY AWAY FROM CHILDREN. HAZARDS INCLUDE CONSUMPTION OF SAWBLADES, NUCLEAR FUEL CONSUMPTION, AND SUDDEN EXTREME TANNING BROUGHT ON BY RADIATION EXPOSURE.")
```

Comment:
How to give players an aneurysm in 20 easy quotes.

Reply:
Understood, adding more quotes now.